Makefile.toml

```diff
@@ -251,16 +245,14 @@ private = true
 script_runner = "bash"
 script = [
 '''
-export PARAMS=$@;
-[[ -z ${PARAMS} ]] && export PARAMS=$(ls ./api_tests/e2e/rfc003/);
-export ROOT=$(pwd);
-for GLOB_DIR in ${PARAMS}; do
-    cd "$ROOT/api_tests/e2e/rfc003/"
-    for DIR in ${GLOB_DIR}; do
-        cd "$ROOT";
-        [[ -d "./api_tests/e2e/rfc003/${DIR}" ]] && ./api_tests/npm_wrapper.sh "./e2e/rfc003/${DIR}"
-    done;
+glob="${1:-*}";
+for dir in api_tests/e2e/rfc003/$glob; do
+    if test -d "$dir"; then
+        files="$files "$dir/*.js;
```

Comment:
❓ does this still work fine if there is an empty directory in `rfc003` ? Or do you get an error "no file found" because no file matches the glob `$dir/*.js`?

Reply:
It works fine.

Reply:
:)